components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -171,7 +171,9 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
     const isPrebuilt = WithPrebuild.is(this.state.workspace?.context);
     let phase = StartPhase.Preparing;
     let title = undefined;
-    let statusMessage = <p className="text-base text-gray-400">Preparing workspace …</p>;
+    let statusMessage = !error
+      ? <p className="text-base text-gray-400">Preparing workspace …</p>
+      : <p className="text-base text-red-500">{error.message}</p>;
```

Comment:
**question**: Is it possible to remove the prefix _Request startWorkspace failed with message_ as it seems unessesary? DO we have more error types we need to surface here or attach with a specific loading phase?

**thought**: The message _No workspace with id 'oops' found._ seems sufficient. No?

Reply:
That's an excellent point. Unfortunately, I'm not sure where this `Request startWorkspace failed with message` part comes from (I assume it's from the "JSON RPC" library we use to make these requests, which would make it difficult to change it).

The rest of the error message comes from here:
https://github.com/gitpod-io/gitpod/blob/40112700f95fab262b896b60b2f0e41b419df710/components/server/src/workspace/gitpod-server-impl.ts#L676

There are indeed other possible error types (e.g. pretty much every time `throw` is used in that file).

If we want to make error messages nicer, I guess we'll need to modify them where they originate from, since we can't really do it in the dashboard.

Reply:
Interessant! Feel free to open a follow up issue fo this. We could tackle this in a future iteration. What do you think?

Reply:
Good idea! Will do. I agree that we should iterate on & improve our error messages.